src/airflow/dags/gwas_catalog_preprocess.py

```diff
@@ -5,19 +5,50 @@
 
 import common_airflow as common
 from airflow.models.dag import DAG
-from airflow.operators.python import PythonOperator
 from airflow.providers.google.cloud.hooks.gcs import GCSHook
-from airflow.providers.google.cloud.operators.gcs import GCSListObjectsOperator
 from airflow.utils.task_group import TaskGroup
 
 CLUSTER_NAME = "otg-preprocess-gwascatalog"
 AUTOSCALING = "otg-preprocess-gwascatalog"
 
-RELEASEBUCKET = "gs://genetics_etl_python_playground/output/python_etl/parquet/XX.XX"
-RELEASEBUCKET_NAME = "genetics_etl_python_playground"
-SUMMARY_STATS_BUCKET_NAME = "open-targets-gwas-summary-stats"
-SUMSTATS = "gs://open-targets-gwas-summary-stats/harmonised"
-MANIFESTS_PATH = f"{RELEASEBUCKET}/manifests/"
+# Setting up bucket name and output object names:
+GWAS_CATALOG_BUCKET_NAME = "gwas_catalog_data"
```

Comment:
I moved all configuration outside the actual step definitions to make sure no parameters/output path is obscure. However it would be nice to abstract these paths further because these outputs needs to be kept consistent with parameters the other DAGs use/produce.

Reply:
agree